services/src/unit-test/java/uk/gov/moj/sdt/services/TargetApplicationSubmissionServiceTest.java

```diff
@@ -620,8 +685,10 @@ public Object answer() throws Throwable {
      */
     @Test
     public void processDlqRequestRejected() {
+        Logger logger = (Logger) LoggerFactory.getLogger(TargetApplicationSubmissionService.class);
+        logger.setLevel(DEBUG);
         final String requestStatus = "REJECTED";
-        final String sdtRequestRef = "TEST_1";
+        final String sdtRequestRef = TEST_1;
```

Comment:
The sdtRequestRef variable is redundant and can be removed.  The TEST_1 constant can be used in place of it.